setup.py

```diff
@@ -66,9 +74,315 @@
 }
 requires['full'] = sum(list(requires.values()), [])
 
+
+class ManPageFormatter(argparse.HelpFormatter):
+    # This code was originally distributed
```

Comment:
just a reminder that this one should be moved to some external file.  could well be a local module right in this directory so no import of `datalad` would be needed